apps/ledger-live-mobile/src/components/RequiresBLE/hooks/useEnableBluetooth.ts

```diff
@@ -147,6 +147,10 @@ export function useEnableBluetooth(
     bluetoothServicesState = "unknown";
   }
 
+  if (__DEV__) {
+    bluetoothServicesState = "enabled";
```

Comment:
As discussed, i would not do this as we won't be able to test correctly this hook in dev mode anymore (as it will always return `enabled` for the state). Checking if the device is a proxy where this hook is used is the way to go